Model/Reclaim.php

```diff
@@ -81,12 +83,19 @@ public function reclaim()
      */
     public function getLog()
     {
-        $log = file($this->_klaviyoLogger->getPath());
-        if ($log != "") {
+        try {
+            $log = file($this->_klaviyoLogger->getPath());
+        } catch (\Exception $e) {
+            return array (
+                'message' => 'Unable to retrieve log file with error: ' . $e->getMessage()
+            );
+        }
+        
+        if ($log != ""  && $log != []) {
```

Comment:
is there a cleaner way to do this?
```suggestion
        if ($log != ''  && $log != []) {
```